script/parseTs.js

```
@@ -0,0 +1,72 @@
+            continue;
+        }
+        if (sourceFile.isDeclarationFile) {
+            ts.forEachChild(sourceFile, visit);
+        }
+    }
+
+    function visit(node) {
+        switch (node.kind) {
+            case ts.SyntaxKind.FunctionDeclaration:
+                // const functionDeclaration = node as ts.FunctionDeclaration;
+                const functionDeclaration = node;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)